### PR TITLE
LPS-198027 Use ctEntry to check for deletion change type

### PR DESCRIPTION
**Pull request by noornajj**
https://liferay.atlassian.net/browse/LPS-198027

### CODE REVIEW

**Comment by noornajj** [comment]
ci:test:publications

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java [hunk 1]
```diff
@@ -72,6 +72,11 @@ private String _getChangeTypeLabel(
 		if (ctEntry.getChangeType() == CTConstants.CT_CHANGE_TYPE_ADDITION) {
 			return _language.get(locale, "added");
 		}
+		else if (ctEntry.getChangeType() ==
```
Comment (david-truong) [inline modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java:75]: can you remove the else?

**Review by david-truong — CHANGES_REQUESTED** [review]

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java (hunk 1 above)
Comment (noornajj) [reply modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java]: I updated the commit

**Comment by noornajj** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
This will be fixed by the changes I made in https://github.com/liferay-publications/liferay-portal/pull/755, so I'm closing this one.

**Pull request closed without merging**